docs/vsphere-golden-image.md

```diff
@@ -68,17 +68,18 @@ the network interface and creates the `tools.conf` file.
 
 Install and configure the OpenSSH Server on the Windows virtual machine with automatic startup and key based-authentication.
```

Comment:
@alinaryan,  as per [comment](https://github.com/openshift/windows-machine-config-operator/pull/515#issuecomment-901179106), the **Set up SSH** section is the following step, after creating the Windows VM and configuring VMWare Tools. Both are platform' specific steps for vSphere.